### DISCUSSION TITLE
“View more”   “Show previous replies”

### DISCUSSION THREAD

**Discussion by rock-crusher** [tagyoureit/nodejs-poolController#603] [General]
Hi guys, just curious, when not logged in and reading the Discussions it seems some of the replies are hidden from view? 

Also, when clicking on “View more” at the bottom of the page the page changes and many of the posts are taken away?

I don’t recall this ever happening before, but have noticed it going on the past few weeks, or so it seems.

Why would some replies be hidden from view when not logged in?

Now, when logged in, it seems the “View more” option has gone away and a new “Show previous replies” has taken its place at the top of page. 

Has the way the site operates changed, or… ?

**Comment by tagyoureit** [comment]
I have no idea.  It's now a core feature of Github.  I took a quick look for any release notes but didn't see anything.  There is a repo and docs out there, but no notes of anything recently changed.

**Comment by rock-crusher** [comment]
Interesting, I liked to be able to read the “Discussions” along with **_ALL_** the replies without having to login each time.

Wonder what’s going on, or if that option can somehow be switched back?
